WordPress-VIP-Go/ruleset-test.php

```diff
@@ -15,7 +15,7 @@
  * @package VIPCS\WordPressVIPGo
  */
 
-namespace WordPressVIPGo;
+namespace WordPressVIPMinimum;
```

Comment:
While this file is in the WordPress-VIP-Go directory, I don't think there's any benefit in having a different namespace for this, at least at this stage. It can be renamed later on, if needed, since no-one else should be referencing anything inside this file.